eclipse-formatter.xml

```diff
@@ -280,13 +280,13 @@
         <setting id="org.eclipse.jdt.core.formatter.insert_space_between_empty_parens_in_enum_constant" value="do not insert" />
         <setting id="org.eclipse.jdt.core.formatter.insert_space_between_empty_parens_in_method_declaration" value="do not insert" />
         <setting id="org.eclipse.jdt.core.formatter.insert_space_between_empty_parens_in_method_invocation" value="do not insert" />
-        <setting id="org.eclipse.jdt.core.formatter.join_lines_in_comments" value="false" />
-        <setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false" />
+        <setting id="org.eclipse.jdt.core.formatter.join_lines_in_comments" value="true" />
+        <setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="true" />
```

Comment:
I'd recommend leaving this at 'false' so lines like these don't get squished all together and become harder to read. Completely fine with joining lines in comments:
```
   public SwerveControllerCommand(
            Trajectory trajectory,
            Supplier<Pose2d> pose,
            SwerveDriveKinematics kinematics,
            PIDController xController,
            PIDController yController,
```

Reply:
And to reformat things without a bunch of manual work, just go back to a previous commit (e.g. the current main branch HEAD) with `git checkout <commitID> .`, then unstage the files and re-apply the new formatting.

Reply:
I can also just use `git reset <hash>`, and yeah I'll do that

Reply:
I think after we tune the formatter enough to match our coding style, it won't be needed

Reply:
Current level of tuning _should_ be enough if there aren't any other issues